.gitignore

```diff
@@ -1,5 +1,6 @@
 /lib
 /node_modules
 /npm-debug.log
-/.DS_Store
+.DS_Store
```

Comment:
looks like you added `flow-typed/.DS_Store` before fixing this pattern. Can you `git rm` it?